ze/Makefile.am

```diff
@@ -139,6 +140,8 @@ CLEANFILES += \
 	$(ZE_STATIC_PROBES_INCL) \
 	$(ZE_STATIC_PROBES_SRC)
 
+
+
 EXTRA_DIST += \
```

Comment:
Can you verify if the other `makefile.am` have some empty line?  We just want to be conscient

Reply:
Removed